docs/user/02-configuration.md

```diff
@@ -28,46 +28,49 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
-| **publisher.&#x200b;replicas.&#x200b;max**               | integer               | Max defines maximum number of replicas.                                                                                                                                                                                                                                                                                                    |
-| **publisher.&#x200b;replicas.&#x200b;min**               | integer               | Min defines minimum number of replicas.                                                                                                                                                                                                                                                                                                    |
-| **publisher.&#x200b;resources**                          | object                | Resources defines resources for eventing-publisher-proxy.                                                                                                                                                                                                                                                                                  |
-| **publisher.&#x200b;resources.&#x200b;claims**           | \[\]object            | Claims lists the names of resources, defined in spec.resourceClaims, that are used by this container. This is an alpha field and requires enabling the DynamicResourceAllocation feature gate. This field is immutable. It can only be set for containers.                                                                                                                                                                                                                                     |
-| **publisher.&#x200b;resources.&#x200b;claims.&#x200b;name** (required) | string | Name must match the name of one entry in pod.spec.resourceClaims of the Pod where this field is used. It makes that resource available inside a container.                                                                                                                                                                                 |
-| **publisher.&#x200b;resources.&#x200b;limits**  | map\[string\]\{integer or string\} | Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/                                                                                                                                                                                |
+| Parameter | Type | Description |
```

Comment:
Revert this file to version from main branch before getting approval.

Reply:
Agree :)